odxtools/odxlink.py

```diff
@@ -172,18 +173,31 @@ class OdxLinkDatabase:
     This can resolve ODXLINK references.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, *, use_weakrefs: bool = False) -> None:
```

Comment:
Avoid default value here, otherwise you miss a case where the value is not propagated from database correctly.

Reply:
e.g. when you want to restrict resolving to a given ECU variant.